gradle/libs.versions.toml

```diff
@@ -0,0 +1,83 @@
+###
+# This section is where we declare the versioning and scope for dependencies of
+# the Web5 platform and projects building atop the Web5 platform.
+#
+# Submodules of Web5 are not able to define their own dependency versions
+# because these must all co-exist in the same ClassLoading environment, and
+# therefore have to be aligned across submodules. Thus we declare the versioning
+# requirements here at the platform level.
+#
+# If a submodule needs to introduce a new dependency or upgrade, define that
+# dependency and version here such that other submodules in the build may pick
+# up the same version. This will guarantee that submodule test suites are running
+# in the correct ClassLoading environment aligned with the Web5 platform.
+#
+# More about Gradle Version Catalogs:
+# https://docs.gradle.org/current/userguide/platforms.html
+#
+# Helpful Blog:
+# https://umang91.medium.com/version-catalogs-in-gradle-7-0-816873b59b47
+###
+
+[versions]
+com_amazonaws_aws_kms = "1.12.538"
+com_danubetech_verifiable-credentials = "1.6.0"
+com_fasterxml_jackson_module = "2.14.2"
+com_github_multiformats = "1.1.0"
+com_google_crypto_tink = "1.10.0"
+com_networknt_json_schema_validator = "1.0.87"
+com_nfeld_jsonpathkt = "2.0.1"
+com_nimbusds = "9.34"
+com_squareup_okhttp3 = "4.12.0"
+com_willowtreeapps_assertk = "0.27.0"
+decentralized_identity_did_common_java = "1.9.0"
+dnsjava = "3.5.2"
```

Comment:
`dnsjava` was the `groupId`; vs something like a `groupId:artifactId` of `decentralized-identity:did-common-java` which I've mapped with underscores because of the `.toml` referencing format.